MBBSEmu/HostProcess/MbbsHost.cs

```diff
@@ -1297,22 +1331,54 @@ private void PatchRelocation(MbbsModule module)
             }
         }
 
+        /// <summary>
+        ///     Marks the specified module as "Enabled" and executes the _INIT_ routine
```

Comment:
doesn't seem to execute the __INIT__ routine in this method

Reply:
Incorrect comment on the method, fixed.